scipy/spatial/transform/rotation.py

```diff
@@ -1461,6 +1463,42 @@ def magnitude(self):
         else:
             return angles
 
+    @classmethod
+    def create_group(cls, group, axis='Z'):
+        """Create a 3D rotation group.
+
+        Parameters
+        ----------
+        group: string
+            The name of the group. Must be one of 'I', 'O', 'T', 'Dn', 'Cn',
+            where `n` is a positive integer. The groups are:
+                I: Icosahedral group. Fixes the rotations of an icosahedron.
```

Comment:
Alternative wording for "fixes rotation of an icosahedron"? I think it's OK to just remove it. 

Could you instead add a link to wikipedia https://en.wikipedia.org/wiki/Point_groups_in_three_dimensions (to the main section)? See the syntax [here](https://github.com/scipy/scipy/blob/master/scipy/interpolate/_cubic.py#L137).

Reply:
The wording was from the book by Conway.  I have removed it though.  I have also replaced the reference to the book with the Wikipedia reference.